src/main/scala/ai/starlake/tests/StarlakeTests.scala

```diff
@@ -64,6 +65,19 @@ case class StarlakeTest(
       d.load(conn)
     }
     expectations.foreach(_.load(conn))
+    preTestStatements.foreach { sql =>
```

Comment:
No need to split the query.
The code below works well:
object X {
  def main(args: Array[String]) = {
    val url = "jdbc:duckdb:/Users/hayssams/tmp/x.db"
    val driver = "org.duckdb.DuckDBDriver"
    val properties = new Properties()
    // properties.put("driver", "org.duckdb.DuckDBDriver")
    val connection = DriverManager.getConnection(url, properties)
    val statement = connection.createStatement()
    statement.execute(
      """
         |CREATE TABLE people (id INTEGER, name VARCHAR);
         |INSERT INTO people VALUES (1, 'Mark'), (2, 'Hannes');""".stripMargin
    )
    statement.close()
    connection.close()

  }
}